### ISSUE TITLE
[@bugsnag/plugin-aws-lambda] Error: Invalid configuration plugins should be an array of plugin objects, got [null]

### ISSUE THREAD

**Issue by briete** [bugsnag/bugsnag-js#1584]
### Describe the bug

We are using Lambda + TypeScript(Node14) and `plugin-aws-lambda` for development.

The following steps are embedded in the Lambda function.
https://docs.bugsnag.com/platforms/javascript/aws-lambda/#usage

Then, I got the following error on Lambda.

### Environment
* Bugsnag version: 7.13.2 (js and plugin-aws-lambda)
* Lambda Node.js 14

### Example code snippet 

In TypeScript, it is written as follows.

#### TypeScript

```typescript
import Bugsnag from '@bugsnag/js';
import BugsnagPluginAwsLambda from '@bugsnag/plugin-aws-lambda';

const AppVersion = process.env.APP_VERSION!;
const BugsnagApiKey = process.env.BUGSNAG_API_KEY!;

Bugsnag.start({
    apiKey: BugsnagApiKey,
    appVersion: AppVersion,
    plugins: [BugsnagPluginAwsLambda],
});

export const bugsnagHandler = Bugsnag.getPlugin('awsLambda')!.createHandler();
```

Here is an excerpt from `tsconfig.json`

```json
{
  "compilerOptions": {
    "target": "ES2018",
    "module": "commonjs",
    "lib": ["es2018"],
    "skipLibCheck": true,
    "declaration": true,
    "strict": true,
    "noImplicitAny": true,
    "strictNullChecks": true,
    "noImplicitThis": true,
    "alwaysStrict": true,
    "noUnusedLocals": false,
    "noUnusedParameters": false,
    "noImplicitReturns": true,
    "noFallthroughCasesInSwitch": false,
    "inlineSourceMap": true,
    "inlineSources": true,
    "experimentalDecorators": true,
    "strictPropertyInitialization": false
  }
}

```

After compiling with `tsc` and deploying to Lambda, the code will look like this.

```javascript
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.notify = exports.bugsnagHandler = void 0;
const js_1 = require("@bugsnag/js");
const BugsnagPluginAwsLambda = require("@bugsnag/plugin-aws-lambda");
const AppVersion = process.env.APP_VERSION;
const BugsnagApiKey = process.env.BUGSNAG_API_KEY;
js_1.default.start({
    apiKey: BugsnagApiKey,
    appVersion: AppVersion,
    plugins: [BugsnagPluginAwsLambda.default],
});
exports.bugsnagHandler = js_1.default.getPlugin('awsLambda').createHandler();
```

When I run Lambda, it outputs the following error and does not work properly.

```
WARN	[bugsnag] Error: Invalid configuration
  - plugins should be an array of plugin objects, got [null]
    at generateConfigErrorMessage (/opt/nodejs/node_modules/@bugsnag/node/core/client.js:338:14)
    at Client._configure (/opt/nodejs/node_modules/@bugsnag/node/core/client.js:147:25)
    at new Client (/opt/nodejs/node_modules/@bugsnag/node/core/client.js:61:25)
    at Object.createClient (/opt/nodejs/node_modules/@bugsnag/node/dist/src/notifier.js:53:21)
    at Object.start (/opt/nodejs/node_modules/@bugsnag/node/dist/src/notifier.js:72:31)
    at Object.<anonymous> (/src/lambda/components/monitor/infrastructures/bugsnug/bugsnug-client.ts:12:5)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
2021-11-17T01:53:40.828Z undefined WARN [bugsnag] Error: Invalid configuration - plugins should be an array of plugin objects, got [null] at generateConfigErrorMessage (/opt/nodejs/node_modules/@bugsnag/node/core/client.js:338:14) at Client._configure (/opt/nodejs/node_modules/@bugsnag/node/core/client.js:147:25) at new Client (/opt/nodejs/node_modules/@bugsnag/node/core/client.js:61:25) at Object.createClient (/opt/nodejs/node_modules/@bugsnag/node/dist/src/notifier.js:53:21) at Object.start (/opt/nodejs/node_modules/@bugsnag/node/dist/src/notifier.js:72:31) at Object.<anonymous> (/src/lambda/components/monitor/infrastructures/bugsnug/bugsnug-client.ts:12:5) at Module._compile (internal/modules/cjs/loader.js:1072:14) at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10) at Module.load (internal/modules/cjs/loader.js:937:32) at Function.Module._load (internal/modules/cjs/loader.js:778:12)
```
</details>

If you remove `.default` from `plugins`, it will work correctly

### NG

```javascript
js_1.default.start({
    apiKey: BugsnagApiKey,
    appVersion: AppVersion,
    plugins: [BugsnagPluginAwsLambda.default],
});
```

### OK

```javascript
js_1.default.start({
    apiKey: BugsnagApiKey,
    appVersion: AppVersion,
    plugins: [BugsnagPluginAwsLambda],
});
```

**Comment by luke-belton** [comment]
Hi @briete - thanks for raising this, looks like the plugin is missing a default property on its module export. We're going to investigate and will keep you updated here 👍

**Comment by luke-belton** [comment]
Hi @briete - just to let you know a [fix](https://github.com/bugsnag/bugsnag-js/pull/1599) for this was released in [`v7.14.1`](https://github.com/bugsnag/bugsnag-js/releases/tag/v7.14.1) 🎉
